docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -547,61 +565,68 @@ Contents of file must be at least 32 hex-encoded bytes and not begin with `0x`.
 May be a relative or absolute path.
 See an [example of how to generate this](../../Tutorials/Merge-Testnet.md#prerequisites).
 
-### `engine-rpc-http-port`
+### `engine-rpc-enabled`
 
 === "Syntax"
 
     ```bash
-    --engine-rpc-http-port=<PORT>
+    --engine-rpc-enabled[=<true|false>]
     ```
 
 === "Example"
 
     ```bash
-    --engine-rpc-http-port=8550
+    ---engine-rpc-enabled=true
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_ENGINE_RPC_HTTP_PORT=8550
+    BESU_ENGINE_RPC_ENABLED=true
     ```
 
 === "Configuration file"
 
     ```bash
-    engine-rpc-http-port=8550
+    engine-rpc-enabled=true
     ```
 
-The listening port for the Engine API calls (`ENGINE`, `ETH`) for JSON-RPC over HTTP.
+Enables or disables the Engine JSON-RPC service.
+The default is `false`.
+
+!!! important
 
-### `engine-rpc-ws-port`
+    You must enable the Engine API to use Besu as an [execution client](../../Concepts/Merge.md#execution-clients) on
+    Ethereum Mainnet.
+
```

Comment:
we should remove/rewrite this to indicate we are deferring to TTD for enabling engine-rpc-api